package.json

```diff
@@ -157,7 +157,7 @@
     "eslint": "bash -c 'opts=$@ ; set -f; if [[ $# -lt 1 ]]; then opts=\"{src/js,plugins,packages}/**/*.js\"; fi ; eslint --quiet ${opts}' 0",
     "cypress": "cypress run",
     "stylelint": "bash -c 'opts=$@ ; set -f; if [[ $# -lt 1 ]]; then opts=\"{src/styles,plugins,packages}/**/*.less\"; fi ; stylelint ${opts}' 0",
-    "conventional-changelog-lint": "conventional-changelog-lint",
+    "commitlint": "./node_modules/.bin/commitlint",
```

Comment:
(why) do we need this? feels like a workaround for something we should fix in a different way 🤔

Reply:
Do you mean that in the `post-commit` we should just directly call call the commitlint binary, rather than add it to the package.json as a script?

Reply:
No, I was wondering why we were able to call the old script without `./node_modules/.bin/`

Reply:
@juliangieseke 
This is not a workaround but a good nodejs practice to not target global module installs.

Note that we are also using for other npm commands, so nothing new.

Reply:
@weblancaster do you have any references for this? AFAIK npm prepends `./node_modules/.bin/` to your `$PATH` variable - if it is configured to do so. 

I remember reading something about absolute paths and that they should be avoided because npm might be set up differently on certain machines. 🤔 
so let me ask again: what is the reason you decided to write it like so? did you try to fix a conflict with a globally installed version?

Reply:
```

Reply:
@juliangieseke you just experienced one of the problems when pointing to global modules. As you pointed out your version is `5.2.8` but the expected version is `6.1.3`. Which `npm run commitlint -- --version` is correct.

Reply:
Yes, agree with @weblancaster. We all want to be using the same version of packages, and being explicit about the path in `package.json` does this.

Reply:
There are many references you can use

http://2ality.com/2016/01/locally-installed-npm-executables.html
https://firstdoit.com/no-need-for-globals-using-npm-dependencies-in-npm-scripts-3dfb478908

Reply:
Ok, looks like we were talking about different things. 🤔
My only question is, why you wrote `./node_modules/.bin/commitlint` instead of just `commitlint` in the package.json. I’m not saying we should use a globally installed version!

From your second link:
> But npm has another trick up its sleeve. When evaluating scripts, npm adds node_modules/.bin/ to the PATH environment variable. That means you can forego the folder prefix entirely!

Reply:
Yes npm automatically looks at the local bin but let's keep it safer and strictly point to local especially because we are already using that way.

Reply:
As we pointed out in our last retro: "because we are already using that way" isnt a good reason to do something. I'd appreciate if we